connector/src/main/java/org/geysermc/connector/utils/WebUtils.java

```diff
@@ -132,4 +136,41 @@ private static String connectionToString(HttpURLConnection con) throws IOExcepti
 
         return content.toString();
     }
-}
+
+    /**
+     * Get the server log file and uploads it to mclo.gs
+     *
+     * @param log File to fetch
+     */
+    public static String mcLogs(Path log) throws IOException {
+        String mcversion = "unknown";
+        String userAgent = "unknown";
+        String version = "unknown";
+        //connect to api
+        URL url = new URL("https://api.mclo.gs/1/log");
+        URLConnection con = url.openConnection();
+        HttpURLConnection http = (HttpURLConnection) con;
+        http.setRequestMethod("POST");
+        http.setDoOutput(true);
+        //convert log to application/x-www-form-urlencoded
+        String content = "content=" + new BufferedReader(new InputStreamReader(Files.newInputStream(log.toRealPath()))).lines().collect(Collectors.joining("\n"));
```

Comment:
```suggestion
        String content = "content=" + new BufferedReader(new InputStreamReader(Files.newInputStream(log.toRealPath()))).lines().collect(Collectors.joining());
```

Since its json, newline chars don't have to be added